### PR TITLE
Fix/tao 3741 rubric block style

### PR DESCRIPTION
**Pull request by jsconan**
Related to: https://oat-sa.atlassian.net/browse/TAO-3741

Fix the issue occurring when a rubric block contains a feedback block and some extended style is applied.

To do so only the actual content is mapped to the editor, instead of the feedback block element, and a wrapper is added around the content if needed.

### CODE REVIEW

**Comment by jsconan** [comment]
@bugalot please do functional testing
@krampstudio can you make a code review? Also, if you have time, can you retro review this other big [PR](https://github.com/oat-sa/extension-tao-testqti/pull/720) that has not been reviewed by the frontend team?

File: views/js/controller/creator/views/rubricblock.js [hunk 1]
```diff
@@ -82,10 +77,41 @@ define([
...
-             * Update the HTML editor
+             * Ensures an html content is wrapped by a container tag.
+             * @param {String} html
+             * @returns {String}
              */
-            function updateContent() {
-                var html = Dom2QtiEncoder.encode(rubricModel.content);
+            function ensureWrap(html) {
+                html = (html || '').trim();
+                if (html.charAt(0) !== '<' && html.substr(3) !== '<br') {
```
Comment (krampstudio) [inline views/js/controller/creator/views/rubricblock.js]: Why only `<br>` you can have the situation with any inline tag, no ? So you can have 
`<a href="#">foo</a> some other text`

File: views/js/controller/creator/views/rubricblock.js [hunk 2]
```diff
@@ -60,13 +60,8 @@ define([
...
         setUp: function setUp(modelOverseer, rubricModel, $rubricBlock) {
             //we need to synchronize the ck elt with an hidden elt that has data-binding
-            var $rubricBlockBinding = $('.rubricblock-binding', $rubricBlock);
             var $rubricBlockContent = $('.rubricblock-content', $rubricBlock);
-            var syncRubricBlockContent = _.throttle(function () {
-                $rubricBlockBinding
-                    .html($rubricBlockContent.html())
-                    .trigger('change');
-            }, 100);
+            var syncRubricBlockContent = _.throttle(editorToModel, 100);
```
Comment (krampstudio) [inline views/js/controller/creator/views/rubricblock.js]: `_.debounce` can be more accurate than `throttle` for this situation

**Comment by jsconan** [comment]
Changes have been made.

**Comment by krampstudio** [comment]
The code is ok for me.

**Comment by krampstudio** [comment]
@bugalot is the PR ready ?

**Comment by bugalot** [comment]
@krampstudio I am currently reviewing it in terms of functionnality. Will merge when OK.

**Pull request merged**